.yamllint.yml

```
@@ -0,0 +1,25 @@
+  brackets:
+    max-spaces-inside: 1
+  indentation:
+    spaces: 2
+    indent-sequences: true
+
+ignore:
+  - node_modules/
+  - .git/
+  - dist/
+  - build/
+  - target/
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)